test/sp.resource.organization_test.js

```diff
@@ -0,0 +1,280 @@
+"use strict";
+
+var common = require('./common');
+var sinon = common.sinon;
+
+var Account = require('../lib/resource/Account');
+var CustomData = require('../lib/resource/CustomData');
+var DataStore = require('../lib/ds/DataStore');
+var Group = require('../lib/resource/Group');
+var IdSiteModel = require('../lib/resource/IdSiteModel');
+var Organization = require('../lib/resource/Organization');
+var OrganizationAccountStoreMapping = require('../lib/resource/OrganizationAccountStoreMapping');
+
+/*jshint -W030 */
+describe('Resources: ', function () {
+  describe('Organization resource', function () {
+    var dataStore = new DataStore({client: {apiKey: {id: 1, secret: 2}}});
+
+    describe('create account', function () {
+      describe('if accounts are set', function () {
```

Comment:
I feel that this part suite needs to be restructure liked this, so that it describes the features and functions under test:

```
create account
  if called without options
    it should post the account
  if called with options
    it should post the account with options
```